API/NoDocs/Microsoft.Extensions.Compliance.Redaction/IRedactionBuilder.cs

```diff
@@ -0,0 +1,13 @@
+// Assembly 'Microsoft.Extensions.Compliance.Abstractions'
+
+using Microsoft.Extensions.Compliance.Classification;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Extensions.Compliance.Redaction;
+
+public interface IRedactionBuilder
+{
+    IServiceCollection Services { get; }
+    IRedactionBuilder SetRedactor<T>(params DataClassification[] classifications) where T : Redactor;
```

Comment:
Set vs Add?

Reply:
Correct, it is setting the redactor to use for a particular data class. It's not a list of redactors per data class, there is just one.